tools/mock-service-host/src/mid/responser.ts

```diff
@@ -355,6 +355,9 @@ export class ResponseGenerator {
             }
             this.swaggerMocker.patchExampleResponses(example, liveRequest)
         }
+        if (liveRequest.query?.[LRO_CALLBACK] === 'true') {
+            this.setStatusToSuccess(example.responses[statusCode].body)
+        }
```

Comment:
Which situation will fall into this condition but not be handled by the following logic?

Reply:
Discussed offline.